src/libraries/System.Net.WebSockets.Client/tests/ConnectTest.Http2.cs

```diff
@@ -40,13 +50,15 @@ public async Task ConnectAsync_VersionNotSupported_Throws()
             async server =>
             {
                 Http2LoopbackConnection connection = await server.EstablishConnectionAsync(new SettingsEntry { SettingId = SettingId.EnableConnect, Value = 0 });
-            }, new Http2Options() { WebSocketEndpoint = true }
+            }, new Http2Options() { WebSocketEndpoint = true, UseSsl = false }
             );
         }
 
-        [Fact]
+        [Theory]
+        [InlineData(false)]
+        [InlineData(true)]
         [SkipOnPlatform(TestPlatforms.Browser, "Self-signed certificates are not supported on browser")]
```

Comment:
If you don't use SSL then this skip message doesn't make sense now, does it?
I suppose we can't run this test on Browser because Loopback server is not supported, but I might be wrong...

Reply:
No, LoopbackServer should work, it uses without that condition in http tests. I separate SSL tests to run under condition